game/src/index.ts

```diff
@@ -69,14 +80,58 @@ const write: PlayerMove<GS, WritePayload> = {
     }
 
     board.answers = {};
-
+    board.playerVotes = {};
+    const allAnswers: string[] = [];
     for (const [userId, playerboard] of Object.entries(playerboards)) {
+      let checkedAnswers: string[] = [];
       const playerAnswers = playerboard.answers[board.round];
-      board.answers[userId] = playerAnswers;
+      board.answers[userId] = playerAnswers.map((a) => {
+        if (
+          playerAnswers.filter(
+            (answer) => answer.toLowerCase() === a.toLowerCase(),
+          ).length > 1
+        ) {
+          const valid = checkedAnswers.includes(a) ? false : true;
+          checkedAnswers.push(a);
+          return {
+            answer: a,
+            valid: valid,
+          };
+        } else {
+          return {
+            answer: a,
+            valid: true,
+          };
+        }
+      });
+
+      allAnswers.push(...playerAnswers);
+    }
+    allAnswers.forEach((a) => {
+      board.playerVotes[a] = {};
+      for (const [userId] of Object.entries(playerboards)) {
+        board.playerVotes[a][userId] = 1;
+      }
+    });
+  },
+};
+
+const vote: PlayerMove<GS, VotePayload> = {
+  executeNow({ board, playerboard, payload }) {
+    if (board.playerVotes[payload.answer][payload.userId] === 1) {
+      board.playerVotes[payload.answer][payload.userId] = 0;
+    } else {
+      board.playerVotes[payload.answer][payload.userId] = 1;
```

Comment:
I'm jumping ahead a little here with the logic getting closer to supporting 2-pointers.

```suggestion
    const { categoryIndex, userId: answerUserId } = payload;
    const currentVote = board.votes[categoryIndex][answerUserId][userId];
    // TODO `maxPoints` should be calculated from the answer.
    const maxPoints = 1;
    const newVote = (currentVote + 1) % (maxPoints + 1);
    board.votes[categoryIndex][answerUserId][userId] = newvote;
```

*EDITED: added the other userId as we need two!*